queries.sql

```diff
@@ -116,4 +116,71 @@ SELECT owners.full_name, COUNT(animals.id) AS num_animals
 FROM owners
 LEFT JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.id
-ORDER BY num_animals DESC;
\ No newline at end of file
+ORDER BY num_animals DESC;
+
+-- Who was the last animal seen by William Tatcher?
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230588391-16bb6778-89f2-4af1-88d9-bf41c171dee9.png)

- [ ] It would be nice if you replace your name with Author1 to make your readme file descriptive and professional 😊